GodelTech.Microservices.Core.sln

```diff
@@ -7,9 +7,13 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GodelTech.Microservices.Cor
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GodelTech.Microservices.Core.Tests", "test\GodelTech.Microservices.Core.Tests\GodelTech.Microservices.Core.Tests.csproj", "{00000000-0000-0000-0000-000000000002}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "GodelTech.Microservices.IntegrationTests", "test\GodelTech.Microservices.IntegrationTests\GodelTech.Microservices.IntegrationTests.csproj", "{5BA960FD-8A69-4303-B29C-7AABDCC14A00}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GodelTech.Microservices.IntegrationTests", "test\GodelTech.Microservices.IntegrationTests\GodelTech.Microservices.IntegrationTests.csproj", "{5BA960FD-8A69-4303-B29C-7AABDCC14A00}"
```

Comment:
@DiamondDragon This Project type guid was changed because of new project system please see. https://github.com/dotnet/project-system/blob/master/docs/opening-with-new-project-system.md. Do I need to revert this change?